Shaders/HSLShift.fx

```diff
@@ -0,0 +1,209 @@
+////////////////////////////////////////////////////
+// HSL Processing Shader                          //
+// Author: kingeric1992                           //
+////////////////////////////////////////////////////
+
+#include "ReShade.fxh"
+
+// UI ELEMENTS /////////////////////////////////////
+////////////////////////////////////////////////////
+  uniform bool EnableHSL <

```

Comment:
This seems a bit superfluous, considering you can just disable the effect instead (which will give you better performance too).

Reply:
Feel free to edit this out, or I can. Whichever you prefer. It's an oversight from when I had this shader combined into a larger one.